source/pkcore/abilitypages/chi/highjump.rst

```diff
@@ -0,0 +1,60 @@
+.. highjump:
+###########
+HighJump
+###########
+
+Description
+###########
+
+HighJump is a Chi Ability.
+
+HighJump allows the Chiblocker to launch themselves high in the air, letting them dodge and surpass obstacles.
+
+
+Usage
+#####
+
+USAGE GIF TO BE ADDED
+
+Function 1: [Left Click]
+
+
+.. tip:: Be cautious of when you HighJump, it gives benders a prime opportunity to strike you.
+
+
+Permissions
+###########
+Below is a table that outlines the permission nodes for this ability.
+
++-------------------------------------+-------------------------------+---------+
+| Permission                          | Function                      | Default |
++-------------------------------------+-------------------------------+---------+
+| bending.ability.HighJump            | Grants access to HighJump     | default |
```

Comment:
Maybe change the default here to be an emoji? :heavy_check_mark: perhaps?

Reply:
I'd rather not as it would mean going back and changing it everywhere else we have permissions. Also no guarantee that the emoji would look the same on everyone's browser and that might cause issues with mobile viewers. I know its not likely but I'd rather not risk it.